packages/app/components/create.tsx

```diff
@@ -463,65 +480,46 @@ function Create() {
             )}
           />
         </View>
-      </CreateScrollView>
 
-      <View tw="mt-8 w-full px-4">
-        {shouldShowSignMessage ? (
-          <View tw="px-2">
-            {!isSignRequested ? (
+        <View tw="mt-8 w-full px-4">
+          {shouldShowSignMessage ? (
+            <View tw="px-2">
               <Text tw="text-center text-lg dark:text-gray-400">
                 We need a signature in order to complete minting. This won't
                 cost any gas.
               </Text>
-            ) : null}
+              <Button
+                tw={`mt-4 ${isSignRequested ? "opacity-60" : ""}`}
+                size="regular"
+                variant="primary"
+                disabled={isSignRequested}
+                onPress={() => {
+                  signTransaction(signMessageData.data);
+                }}
+              >
+                {isSignRequested ? "Signing..." : "Sign Message"}
```

Comment:
Maybe we could simplify `Sign Message` to `Sign`?

Reply:
Would be the case for `Sign Message` in `drop` and also `Sign the message` in `login`